contracts/utils/Selector.sol

```diff
@@ -10,7 +10,7 @@ contract Selector {
     function calculateXcertSelector() public pure returns (bytes4) {
       Xcert i;
       return i.mint.selector
-         ^ i.tokenProof.selector
+      //   ^ i.tokenProof.selector
```

Comment:
Let's not comment the code - if we don't need it, we remove it. Git history is our friend for all other cases.